From 250e82252b53991e2902b292cfa6029ab28a10fb Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Pavel=20B=C5=99ezina?= <pbrezina@redhat.com>
Date: Mon, 15 Oct 2018 12:46:35 +0200
Subject: [PATCH 77/83] sudo: use correct sbus interface

Internal dbus interfaces were renamed to shorter names in sbus2.

Resolves:
https://pagure.io/SSSD/sssd/issue/3854

Reviewed-by: Jakub Hrozek <jhrozek@redhat.com>
---
 src/responder/sudo/sudosrv_dp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/responder/sudo/sudosrv_dp.c b/src/responder/sudo/sudosrv_dp.c
index 2c6b26e..78dd296 100644
--- a/src/responder/sudo/sudosrv_dp.c
+++ b/src/responder/sudo/sudosrv_dp.c
@@ -66,7 +66,7 @@ sss_dp_get_sudoers_msg(TALLOC_CTX *mem_ctx,
 
     msg = dbus_message_new_method_call(bus_name,
                                        SSS_BUS_PATH,
-                                       "org.freedesktop.sssd.dataprovider",
+                                       "sssd.dataprovider",
                                        "sudoHandler");
     if (msg == NULL) {
         DEBUG(SSSDBG_CRIT_FAILURE, "Out of memory?!\n");
-- 
2.9.5

